SVGKit-iOS.xcodeproj/project.pbxproj

```diff
@@ -1161,6 +1161,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = NO;
```

Comment:
Weird that this was needed, as I can't seen that there are any strings files in this project 🤔 unless it was referring to strings in the Info.plist for the demo project 🤷

Reply:
It's just a standard Xcode-recommended warning that it thinks should be applied everywhere.
The default was to set it to YES, but I don't want that kicking out noise about "violations" so I changed it to NO and it seems happy with that.